src/lib/logger/fake/logger.ml

```diff
@@ -145,14 +146,19 @@ end]
 
 let metadata t = t.metadata
 
-let create ?metadata:_ ?(id = "default") () =
-  { null = false; metadata = Metadata.empty; id }
+let create ?metadata:_ ?(id = "default") ?itn_features:_ () =
```

Comment:
It should be okay if we leave only the line above changed with the rest of file remaining the same it was

Reply:
Implementation of `fake.ml` is somewhat confusing, see this PR which brings some clarity: #16039

I.e. we don't really need to maintain the same type `t` as in `native.ml`.